templates/authenticate.php

```diff
@@ -7,7 +7,14 @@
 <form method="post">
 	<fieldset class="warning">
 		<?php if (!$_['wrongpw']) { ?>
-			<div class="warning-info"><?php p($l->t('This conversation is password-protected')); ?></div>
+			<div class="warning-info">
+				<?php p($l->t('This conversation is password-protected.')); ?>
+				<?php if ($_['showBruteForceWarning']) { ?>
+					<?php p($l->t('We have detected multiple invalid password attempts from your IP. Therefore your next attempt is throttled up to 30 seconds.')); ?>
+				<?php } ?>
+			</div>
+		<?php } elseif ($_['showBruteForceWarning']) { ?>
+			<div class="warning-info"><?php p($l->t('We have detected multiple invalid password attempts from your IP. Therefore your next attempt is throttled up to 30 seconds.')); ?></div>
```

Comment:
Maybe it would be better to add the warning after the `The password is wrong. Try again.` rather than replace it, similar to [what is done in the login page](https://github.com/nextcloud/server/blob/a1aaaaa0c86c002a0a406464724b8a0236fe8406/core/src/components/login/LoginForm.vue#L94-L106).

Reply:
Done